src/py/flwr/server/driver/driver.py

```diff
@@ -45,6 +45,11 @@ def set_run(self, run_id: int) -> None:
     def run(self) -> Run:
         """Run information."""
 
+    @property
+    @abstractmethod
+    def message_ids(self) -> list[str]:
```

Comment:
Should we keep the same signature as `push_messages`?

```suggestion
    def message_ids(self) -> Iterable[str]:
```